etc/kayobe/kolla/config/monasca/persister.conf

```diff
@@ -1,2 +1,12 @@
 [kafka_alarm_history]
 enabled = False
+
+# This prevents the persister jamming on bad metrics. Metrics which are
+# skipped are logged in the persister log file.
+[repositories]
+ignore_parse_point_error = True
+
+# The default is 2, but empirically this can overload InfluxDB when a backlog
```

Comment:
To be precise, the default **in kolla-ansible** is 2 (configured via ``monasca_metric_pipeline_threads``).